darwin/config.py

```diff
@@ -54,12 +64,19 @@ def get(self, key: Union[str, List[str]], default: Optional[Any] = None) -> Any:
                 key = keys
 
     def put(self, key: Union[str, List[str]], value: Any, save: bool = True) -> None:
-        """Sets value for specified key
-
-        Args:
-        - key: the key where the value is going to be stored.
-        It can be formatted as a simple string, or as a path/like/string to fetch nested values.
-        - value: the value to be set"""
+        """
+        Sets the value for the specified key.
+
+        Parameters
+        ----------
+        key: Union[str, List[str]]
+            The key where the value is going to be stored.
+            It can be formatted as a simple string, or as a path/like/string to fetch nested values.
+        value: Any
+            The value to be stored.
+        save: bool
+            If True, persists the value in the FileSystem. Defaults to 'True'.
```

Comment:
I think backticks are preferred?
```suggestion
            If True, persists the value in the FileSystem. Defaults to `True`.
```